src/plugins/plugin.legend.js

```diff
@@ -440,12 +507,28 @@ var Legend = Element.extend({
 				};
 			}
 
-			var itemHeight = fontSize + labelOpts.padding;
+			var itemHeight = maxHeight;
 			helpers.each(me.legendItems, function(legendItem, i) {
-				var textWidth = ctx.measureText(legendItem.text).width;
+				var textWidth;
+				var boxTopOffset;
+
+				if (helpers.isArray(legendItem.text)) {
+					var widths = legendItem.text.map(function(textLine) {
```

Comment:
you can inline this and remove the variable declaration

Reply:
Done